optuna/integration/chainermn.py

```diff
@@ -124,6 +124,11 @@ def optimize(
         the absence of ``n_jobs`` argument.
         """
 
+        if n_trials:
```

Comment:
I think we’ve better put the warning logs in the visualization functions rather than in the optimize method.  Otherwise, it impacts even the user not using visualization features.

Reply:
hi but even plot_intermediate_values don't have n_trials parameter anymore

Reply:
Agree with @g-votte. I think that, within visualization functions, we can get the number of trials in a study via `len(study.trials)`.